### PR TITLE
install: nuthin but tarball (and rpm)

### PR DESCRIPTION
**Pull request by dweomer**
Remove use of makeself. Our installations options are RPM or tarball
extraction to /usr/local. Also:

- update install.sh
- update installation section of readme

Partially addresses #295

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,37 +19,55 @@ RKE2 is a fully compliant Kubernetes distribution, with the following changes:
...
 
 ```sh
-INSTALL_RKE2_VERSION=v0.0.1-alpha.7 ./install.sh
+# Y.O.L.O.
+curl -fsSL https://raw.githubusercontent.com/rancher/rke2/master/install.sh | sudo sh -
 ```
 
-### CIS Mode
-
-If you plan to run your RKE2 cluster in CIS mode, the installation script offers the ability to install RKE2 in preparation for this. Use the `INSTALL_RKE2_CIS_MODE` environment variable. This will configure the system for RKE2 to run in CIS compliant mode. This includes the creation of a user "etcd", updates a number of kernel parameters, and installs a systemd or openrc unit file that passes the CIS mode flag to the binary (`--profile=cis-1.5`). Please reference the installation script (`install.sh`) for CIS specifics.
```
Comment (briandowns) [inline README.md:34]: Is this information going to be put somewhere else? I think it has value to the user.

**Review by briandowns — APPROVED** [review]
🚢 🇮🇹

**Comment by dweomer** [comment]
```sh
$ tar tvf dist/artifacts/rke2.linux-amd64.tar.gz 
drwxrwxr-x jacob/jacob       0 2020-09-14 16:32 bin/
-rwxrwxr-x jacob/jacob 156710656 2020-09-14 16:32 bin/rke2
-rwxrwxr-x jacob/jacob      1828 2020-09-14 16:27 bin/rke2-killall.sh
-rwxrwxr-x jacob/jacob      1201 2020-09-14 16:27 bin/rke2-uninstall.sh
drwxrwxr-x jacob/jacob         0 2020-09-14 16:27 lib/
drwxrwxr-x jacob/jacob         0 2020-09-14 16:27 lib/systemd/
drwxrwxr-x jacob/jacob         0 2020-09-14 16:27 lib/systemd/system/
-rw-rw-r-- jacob/jacob       659 2020-09-14 16:27 lib/systemd/system/rke2-agent.service
-rw-rw-r-- jacob/jacob       663 2020-09-14 16:27 lib/systemd/system/rke2-server.service
drwxrwxr-x jacob/jacob         0 2020-09-14 16:32 share/
drwxrwxr-x jacob/jacob         0 2020-09-14 16:32 share/rke2/
-rw-rw-r-- jacob/jacob     10174 2020-09-14 16:32 share/rke2/LICENSE.txt
drwxrwxr-x jacob/jacob         0 2020-09-14 16:27 etc/
drwxrwxr-x jacob/jacob         0 2020-08-26 17:46 etc/sysctl.d/
-rw-rw-r-- jacob/jacob       115 2020-08-26 17:46 etc/sysctl.d/rke2-cis.conf
```

**Review by galal-hussein — APPROVED** [review]
LGTM

**Review by brandond — APPROVED** [review]

**Comment by dweomer** [comment]
@davidnuzik we will want a documentation review for these changes. They will at the very least alter QA testing procedures.

File: README.md [hunk 2]
```diff
@@ -19,37 +19,55 @@ RKE2 is a fully compliant Kubernetes distribution, with the following changes:
...
+RKE2 comes with a convenient installation script that will, by default, install via `yum` on RPM-based systems while unpacking the tarball bundle to `/usr/local` on all others.
 
 ### Quick Start
 
-The `install.sh` script provides a convenient way for installing rke2 server and agent, to install rke2 service:
+To install via the default mechanism for your system:
 
 ```sh
-INSTALL_RKE2_VERSION=v0.0.1-alpha.7 ./install.sh
+# Y.O.L.O.
```
Comment (davidnuzik) [inline README.md:29]: Sorry to turn down the fun dial - but do we really want our public docs to say YOLO?

Comment (dweomer) [reply README.md:29]: Well. It is important to provide a simple, scriptable one-liner but it is also important to acknowledge that it is insecure.

Comment (dweomer) [reply README.md]: Its removed now.

Comment (davidnuzik) [reply README.md]: Thanks

Comment (cjellick) [reply README.md]: eh, im with david unfortunately. this is the source-of-truth for RFed's users/customers. theyre being actively driven to it and can't guarantee everyone will react positively to this. :-/

**Pull request merged**